Toshi/Source/Toshi/TClass.cpp

```diff
@@ -61,14 +37,14 @@ bool __thiscall Toshi::TClass::AttachClassToParent()
 	return false;
 }
 
-Toshi::TClass::TClass(const char* name, TClass* parent, t_CreateTObject create, t_CreateTObjectAtPlace createAtPlace, t_RegisterScriptingAPI scripting, uint32_t version, size_t size)
+Toshi::TClass::TClass(const char* name, TClass* parent, t_CreateTObject create, t_CreateTObjectAtPlace createAtPlace, t_RegisterScriptingAPI scripting, uint16_t verMajor, uint16_t verMinor, size_t size)
```

Comment:
That's ok but i want to keep it as JPOG and de blob does it, i think.

Reply:
but why? isn't this better? 🤔